src/services/MusicService.js

```diff
@@ -62,4 +62,19 @@ export class MusicService {
       .once("value")
       .then(author => author.val());
   }
+
+  static getSongById(songId) {
+    return database
+      .ref(`songs/${songId}`)
+      .once("value")
+      .then(song => song.val());
+  }
+
+  static getAuthorNameFromId(authorsIds) {
```

Comment:
actually you return array of author names - so name of the function should be in plural - eg `getAuthorNamesByIds()`